pkg/koordlet/util/system/cgroup_driver.go

```diff
@@ -193,7 +200,61 @@ var cgroupPathFormatterInCgroupfs = formatter{
 }
 
 // CgroupPathFormatter uses the Systemd cgroup driver by default.
-var CgroupPathFormatter = cgroupPathFormatterInSystemd
+var CgroupPathFormatter = GetCgroupFormatter()
```

Comment:
could you please add a unit test to verify the fix

Reply:
done